Solutions/Marain.Tenancy.Specs/Integration/Features/TenancyClient.feature

```diff
@@ -71,6 +71,21 @@ Scenario: Get children
 	| ChildTenant4 |
 	| ChildTenant5 |
 
+Scenario: Get children when no child tenants exist
+	Given I create a child tenant called "ChildTenant1" for the root tenant
+	When I get the tenant id of the tenant called "ChildTenant1" and call it "ChildTenantId"
+	And I get the children of the tenant with the id called "ChildTenantId" with maxItems 20 and call them "Result"
+	Then there should be no ids in the children called "Result"
```

Comment:
I don't understand what this is saying. What does "the children called <x>" mean?

The wording feels like it says more about the way the test is constructed than what you actually want to verify. I think this is a statement that calling something (not actually named anywhere in this test as far as I can see, but some sort of "get children" operation) should produce an empty result? Or possible a result that's not actually empty but which contains no IDs?

If this read something like "the list of IDs returned by GetChildren should be empty".

I'm wondering if it's possible to remove the `"Result"` entirely from this test. I know that sometimes you need to be able to label results because a single scenario might make multiple calls and you need to be able to distinguish them, but is it possible to add steps that just plug in some default name so that you can word these things more naturally in cases where there is actually just the one?

Reply:
Happy to look at the specs in more detail, but I think that if we want to change the way they are written, we should create a separate issue to track this rather than make it part of this PR. Are you happy with that?

Reply:
Done: #77